docs/libtorch.rst

```diff
@@ -34,3 +34,18 @@ To produce libtorch.a rather than libtorch.so, set the environment variable `BUI
 To use ninja rather than make, set `CMAKE_GENERATOR="-GNinja" CMAKE_INSTALL="ninja install"`.
 
 Note that we are working on eliminating tools/build_pytorch_libs.sh in favor of a unified cmake build.
+
+Building libtorch without using Python
+--------------------------------------
+
+You can build C++ libtorch.so without relying on Python.  For example to build a v1.6.0 Release version that uses (system-installed) CUDA and CUDDN, and install it in the directory specified by CMAKE_INSTALL_PREFIX below, you can use
+::
+   git clone https://github.com/pytorch/pytorch.git
+   cd pytorch
+   git checkout v1.6.0
+   git clean -fdx
+   git submodule update --init --recursive
+   mkdir ../pytorch-build
+   cd ../pytorch-build
+   cmake -DBUILD_SHARED_LIBS:BOOL=ON -DCMAKE_BUILD_TYPE:STRING=Release -DUSE_CUDA:BOOL=ON -DUSE_CUDNN:BOOL=ON -DCMAKE_INSTALL_PREFIX:PATH=../pytorch-install ../pytorch
```

Comment:
No need to specify configuration option that should be auto-detected (CMake will try to build with CUDA/CUDNN by default if it is available on the system search path)
```suggestion
   cmake  -DCMAKE_BUILD_TYPE=Release -DPYTHON_EXECUTABLE=`which python3` -DCMAKE_INSTALL_PREFIX:PATH=../pytorch-install ../pytorch
```

Reply:
I explicitly listed some options with their default values.  That makes it easier for the user to see how to make changes, to other than default values.  I can add -DPYTHON_EXECUTABLE=`which python3`

Reply:
Please see the most recent commit.

Reply:
Can you please elaborate, how do you decide, which options to include and which aren't?
Wouldn't it be easier to read/understand if default instructions would be as concise as possible and then have an additional paragraphs covering useful configurations, for example:
 - How to build  PyTorch with CUDA support if default gcc is too new (by switching to clang or using cuda-gcc from rpmfusion), i.e. by specifing: `-DCMAKE_CXX_COMPILER=/usr/bin/cuda-g++ -DCMAKE_C_COMPILER=/usr/bin/cuda-gcc`
- How to build PyTorch with address sanitizer (-DUSE_ASAN) or in RelWithDebInfo mode, which gives one a performant runtime but at the same time a powerful features such as symbolicated backtrace, line information and even some frame variable information.